bioblend/_tests/pytest_galaxy_test_wrapper.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+import os
+import sys
+
+try:
+    import pytest
```

Comment:
I don't want to hold you up - definitely feel free to merge before I get that done. I'll create an issue and assign myself.